### ISSUE TITLE
Explorer Update

### ISSUE THREAD

**Issue by jmdelahanty** [Tyelab/bruker_control#79]
Having a way for running bruker_control to find planes for different animals during an `expedition` phase that will then execute 500 frames of resonant imaging as well as a couple galvo images averaged 4 times for good measure or something, store them in a particular way, and create imaging metadata that the experimental script can call without loading the XML of these images would be pretty cool. It would be it's own "experiment" type. The basic functionality should be incorporated before the flight manifest update is done and after the second refactor is complete.

**Comment by jmdelahanty** [comment]
The second refactor will likely have to take a backseat for things. Having some of these additional features can be pretty nice and the backend for how to store this sort of information properly needs to be done before this update is in place. Need to make a new issue now...
